python/tvm/relay/op/nn/nn.py

```diff
@@ -1621,3 +1621,7 @@ def bitserial_dense(data,
     """
     return _make.bitserial_dense(data, weight, units, data_bits, weight_bits,
                                  pack_dtype, out_dtype, unipolar)
+
+
+def cross_entropy(predictions, targets):
```

Comment:
This should have a docstring. You should also mention that this is cross-entropy without softmax, as many frameworks equate cross-entropy to cross-entropy from logits

Reply:
@MarisaKirisame can you react on @SWu 's comment (also put it in REGISTER_RELAY_OP section)

Reply:
ping @MarisaKirisame